src/Data/StatDescriptions/gem_stat_descriptions.lua

```diff
@@ -15885,6 +15885,100 @@ return {
 			[1]="minions_are_defensive"
 		}
 	},
+	[558]={
+		lang={
+			English={
+				[1]={
+					[1]={
+						k="per_minute_to_per_second",
+						v=1
+					},
+					limit={
+						[1]={
+							[1]=1,
+							[2]="#"
+						}
+					},
+					text="Deals {0} Base Physical Damage per second"
+				}
+			}
+		},
+		stats={
+			[1]="base_physical_damage_to_deal_per_minute"
+		}
+	},
+	[559]={
+		lang={
+			English={
+				[1]={
+					limit={
+						[1]={
+							[1]="#",
+							[2]="#"
+						}
+					},
+					text="Gain a Blood Charge if this Skill Hits Enemies and none of them Die"
+				}
+			}
+		},
+		stats={
+			[1]="gain_blood_charge_on_enemy_survives"
+		}
+	},
+	[560]={
+		lang={
+			English={
+				[1]={
+					limit={
+						[1]={
+							[1]=1,
+							[2]="#"
+						}
+					},
+					text="Deals {0}% more Damage per Blood Charge"
+				}
+			}
+		},
+		stats={
+			[1]="damage_+%_final_per_blood_charge"
+		}
+	},
+	[561]={
+		lang={
+			English={
+				[1]={
+					limit={
+						[1]={
+							[1]=1,
+							[2]="#"
+						}
+					},
+					text="Costs {0}% more Life per Blood Charge"
+				}
+			}
+		},
+		stats={
+			[1]="costs_life_+%_final_per_blood_charge"
+		}
+	},
+	[562]={
+		lang={
+			English={
+				[1]={
+					limit={
+						[1]={
+							[1]="#",
+							[2]="#"
+						}
+					},
+					text="Loose a Blood Charge when an Enemy Dies while affected by this Skill's Debuff"
```

Comment:
```suggestion
					text="Lose a Blood Charge when an Enemy Dies while affected by this Skill's Debuff"
```

Reply:
Hey, don't mean to be petty or anything, just thought I'd let you know there's a typo here while scrolling through. :) Thanks for your work! Keep it up! :)